index.html

```diff
@@ -865,10 +919,86 @@ <h4>issuance</h4>
 <section id="conformance"></section>
 
 <section>
-<h2>Privacy Considerations</h2>
+  <h2>Privacy Considerations</h2>
+  <p>
+    Verifiable Credentials often contain sensitive information that should be protected 
+    to ensure the privacy and security of individuals. This section outlines some privacy 
+    considerations that implementers and users of this specification should be aware of.
+  </p>
+  <p>
+    Implementers SHOULD note and abide by all privacy considerations called out in the [[VC-DATA-MODEL]].
+  </p>
+  <p>
+    In addition to privacy recommendations noted in the [[VC-DATA-MODEL]] the following 
+    considerations should be explicity observed:
+    <ul>
+      <li>
+        <p>
+          Minimization of data: Verifiable Credentials should only contain the minimum amount 
```

Comment:
This seems like guidance that is not JWT specific, but I think it is fine to add here for now.

We should probably start by advising implementers to follow guidance from:

https://www.rfc-editor.org/rfc/rfc7519#section-12

Reply:
did so above, tahnks!